ui/components/app/signature-request/__snapshots__/signature-request.component.test.js.snap

```diff
@@ -249,14 +249,14 @@ exports[`Signature Request Component render should match snapshot when we are us
             class="box signature-request-data__node box--padding-left-2 box--display-flex box--flex-direction-row"
           >
             <span
-              class="box box--margin-top-1 box--margin-bottom-1 box--margin-left-4 box--flex-direction-row typography typography--p typography--weight-normal typography--style-normal typography--color-text-default"
+              class="box mm-text mm-text--body-md mm-text--font-weight-normal box--margin-left-4 box--flex-direction-row box--color-text-default"
```

Comment:
You may need to update snapshots I was expecting this to use the `mm-box` className?

Reply:
Hey @georgewrmarshall ,
Updated the snapshots.👍
Please look into this.